orchestrator/db/filters/process.py

```diff
@@ -112,16 +96,17 @@ def target_filter(query: SearchQuery, value: str) -> SearchQuery:
 
 
 VALID_FILTER_FUNCTIONS_BY_COLUMN: dict[str, Callable[[SearchQuery, str], SearchQuery]] = {
```

Comment:
Just curious on how this is going to work out when extending queries. Will we be able to easily add additional filter functions, etc.?

Reply:
that would be the idea yea.